crates/avro/src/lib.rs

```diff
@@ -43,14 +45,8 @@ pub enum Error {
 }
 
 /// Map a [`doc::Shape`] and key pointers into its equivalent AVRO schema.
-pub fn shape_to_avro(
-    shape: doc::Shape,
-    key: &[doc::Pointer],
-) -> (apache_avro::Schema, apache_avro::Schema) {
-    (
-        schema::key_to_avro(key, shape.clone()),
-        schema::shape_to_avro(json::Location::Root, shape, true),
-    )
+pub fn shape_to_avro(shape: doc::Shape) -> apache_avro::Schema {
```

Comment:
See [here](https://github.com/estuary/flow/pull/1840/files#diff-8cb17d92f697c0e1519db60f893d21d41cc59c75e2d9872ff522072968357b22R265-R266) for why I made this change. I'm happy to try and find a different way if you feel like you'd rather keep the API of the `avro` crate the same, this was just the most direct path, and Dekaf is the only consumer of this code so far so it was pretty limited in impact